_quarto.yml

```diff
@@ -46,6 +46,7 @@ book:
     - responsible_ai.qmd
     - sustainable_ai.qmd
     - generative_ai.qmd
+    - emerging_hw.qmd
```

Comment:
Could you move this to be after `hw_acceleration.qmd`

Reply:
Take a look at the hw_acceleration contents, there is some stuff in there but I do like the idea of pulling out emerging trends separately, either in the same file or in a new file.